megamek/src/megamek/common/Game.java

```diff
@@ -674,11 +674,11 @@ public GamePhase getPhase() {
     public void receivePhase(GamePhase phase) {
         final GamePhase oldPhase = this.phase;
```

Comment:
should this be remove since oldPhase is no longer used

Reply:
Yes that should be removed. - I saw that after I committed it.